setup.py

```diff
@@ -98,6 +98,11 @@ def pypi_link(pkg_filename):
     long_description_content_type='text/x-rst; charset=UTF-8',
     keywords="CPAN PyPI distutils eggs package management",
     url="https://github.com/pypa/setuptools",
+    project_urls={
```

Comment:
I can't help but feel like this setting is adding a lot of repetition. Now `https://github.com/pypa/setuptools` appears three times in the metadata. Perhaps if this only included the `Documentation` link, I'd feel better than a usage that demands so much repetition. I don't feel strongly about it, though.

Reply:
Happy to adjust this to whatever you like honestly. This was mostly just a placeholder while seeking feedback from the setuptools maintainers as to what they'd want to see added for this set (if anything). I'll scale this back to just the Documentation link here so it's still dogfooded, and then you can add/update any URLs you like in the future obviously.